examples/reference/elements/bokeh/Tiles.ipynb

```diff
@@ -92,11 +100,24 @@
   }
  ],
  "metadata": {
```

Comment:
I don't know why / how to filter out the metadata below. I ran the following jupyter commands but that didn't strip these:

```
jupyter nbconvert --clear-output --ClearMetadataPreprocessor.clear_notebook_metadata=True --ClearMetadataPreprocessor.clear_cell_metadata=True Tiles.ipynb
```

I can fix this manually but wanted to ask if someone can point me to the correct `jupyter nbconvert` command for it.